README.rst

```diff
@@ -22,14 +23,19 @@ reStructuredText for PEPs
 
 Original PEP source should be written in reStructuredText format,
 which is a constrained version of plaintext, and is described in
-PEP 12.  Older PEPs were often written in a more mildly restricted
-plaintext format, as described in PEP 9.  The ``pep2html.py``
-processing and installation script knows how to produce the HTML
-for either PEP format.
+`PEP 12 <https://www.python.org/dev/peps/pep-0012/>`_.
+Older PEPs were often written in a more mildly restricted
+plaintext format, as described in `PEP 9
+<https://www.python.org/dev/peps/pep-0009/>`_.
+The ``pep2html.py`` processing and installation script knows
+how to produce the HTML for either PEP format.
```

Comment:
The only remaining plain text PEP is PEP 9; I'd remove these five lines, maybe keeping the pep2html bit.

Reply:
Ah, didn't realize that, but I can confirm with some quick grepping that every single PEP has a `Content-Type: text/x-rst` header, even PEP 9 itself, which is basically just the original PEP content inside a literal reST code-block, and if present, it is checked by the linter. Funny enough, I can confirm that despite that, plain text is _still_ the default mode of the legacy docutils renderer as removing the header causes it to fall back to the old rendering, despite nothing actually using it—of course, this is not the case for the new PEP 676 one. If and when that PEP is accepted, the field could simply be removed entirely.

In any case, I've removed/revised these lines accordingly. I'll open another PR to update PEP 1 (which still references plain-text PEPs a handful of places), and the template (which appears to be missing the Content-Type line completely), and we can also flip the default, since no plain-text PEPs even exist anymore.

Reply:
(Opened that PR as #2355)

Reply:
PR merged